pennylane/templates/subroutines/single_excitation_op.py

```diff
@@ -0,0 +1,160 @@
+    creation operators and the indices :math:`r` and :math:`p` run over the occupied and
+    unoccupied molecular orbitals, respectively. Using the `Jordan-Wigner transformation 
+    <https://arxiv.org/abs/1208.5986>`_ the fermionic operator defined above can be written 
+    in terms of Pauli matrices, (see Ref. `arXiv:1805.04340 <https://arxiv.org/abs/1805.04340>`_)
+
+    .. math::
+
+        \hat{U}_{pr}^{(1)}(\theta) = \mathrm{exp} \Big\{ \frac{i\theta}{2} 
+        \bigotimes_{a=r+1}^{p-1}\hat{Z}_a (\hat{Y}_r \hat{X}_p) \Big\} 
+        \mathrm{exp} \Big\{ -\frac{i\theta}{2}
+        \bigotimes_{a=r+1}^{p-1} \hat{Z}_a (\hat{X}_r \hat{Y}_p) \Big\}.
+
+    The quantum circuit to exponentiate the tensor product of Pauli matrices entering
+    the latter equation is shown below:
+
+    .. figure:: ../../_static/ucc_se_op.png
+        :align: center
+        :width: 60%
+        :target: javascript:void(0);
+
+    As explained in the paper by Seely *et al.* `arxiv:1208.5986 <https://arxiv.org/abs/1208.5986>`_
+    an :math:`n`-fold tensor product of Pauli-Z matrices requires :math:`2(n-1)` CNOT gates and a
+    single-qubit Z-rotation to exponentiate on a quantum computer. If there are :math:`X` or 
+    :math:`Y` Pauli matrices in the product, the Hadamard (:math:`H`) or :math:`R_x` gate has to 
+    be applied to change to the :math:`X` or :math:`Y` basis, respectively.
+
+    Notice that: 
+
+    #. :math:`\hat{U}_{pr}^{(1)}(\theta)` involves two exponentiations where :math:`\hat{U}_1`,
+       , :math:`\hat{U}_2` and :math:`\hat{U}_\theta` are dfined, for each of them, as follows,
+
+       .. math::
+           [U_1, U_2, U_{\theta})] = \Bigg\{\bigg[R_x(-\pi/2), H, R_z(\theta/2)\bigg], 
+           \bigg[H, R_x(-\frac{\pi}{2}), R_z(-\theta/2) \bigg] \Bigg\}
+
+    #. For a given pair ``[r, p]`` ten single-qubit operations are applied. Notice also that 
+       CNOT gates act only on qubits with indices between ``r`` and ``p``. The operations 
+       performed accross these qubits are shown in dashed lines in the figure above. 
+
+    Args:
+        weight (float): angle :math:`\theta` entering the Z rotation acting on wire ``p``
+        wires (sequence[int]): two-element sequence with the qubit indices ``r, p``
+
+    Raises:
+        ValueError: if inputs do not have the correct format
+    """
+
+    ##############
+    # Input checks
+
+    check_no_variable(wires, msg="'wires' cannot be differentiable")
+
+    wires = check_wires(wires)
+
+    expected_shape = (2,)
+    check_shape(
+        wires,
+        expected_shape,
+        msg="'wires' must be of shape {}; got {}" "".format(expected_shape, get_shape(wires)),
+    )
+
+    expected_shape = ()
+    check_shape(
+        weight,
+        expected_shape,
+        msg="'weight' must be of shape {}; got {}" "".format(expected_shape, get_shape(weight)),
+    )
+
+    check_type(wires, [list], msg="'wires' must be a list; got {}" "".format(wires))
```

Comment:
Are there too many inverted commas in `got {}" ""`. Here and in other cases below?

Reply:
I noticed this is the standard in other templates when calling the utility functions.

Reply:
Oh, that is indeed a typo. I will check the other templates.

Reply:
I know where it came from: There was a line break in the original code, and "blacking" the code automatically removed the line break and led to this funny format.

Reply:
Luckily Python has implicit string concatenation, so this is still valid syntax 🙂